sunbird-es-utils/src/main/resources/mappings/org-mapping.json

```diff
@@ -822,6 +822,22 @@
             "analyzer": "cs_index_analyzer",
             "search_analyzer": "cs_search_analyzer",
             "fielddata": true
+        },
```

Comment:
@karthik-tarento Please remove the ES mapping changes from here and raise a PR against https://github.com/project-sunbird/sunbird-devops/tree/master/ansible/roles/es-mapping/files/